lib/invoicing/taxable.rb

```diff
@@ -250,7 +250,12 @@ def write_attribute(attribute, value) #:nodoc:
         @taxed_or_untaxed[attribute] = :untaxed
         @taxed_attributes[attribute] = nil
       elsif attribute =~ /^(#{attr_regex})_taxed$/
-        @attributes[attribute] = value
+        if ActiveRecord::VERSION::MAJOR == 4 && ActiveRecord::VERSION::MINOR >= 2
```

Comment:
searched if Rails provides some methods, but couldn't find method which returns major and minor versions.